provisioner/converge/provisioner.go

```diff
@@ -87,6 +90,10 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 		p.config.ExecuteCommand = "cd {{.WorkingDirectory}} && {{if .Sudo}}sudo {{end}}converge apply --local --log-level=WARNING --paramsJSON '{{.ParamsJSON}}' {{.Module}}"
 	}
 
+	if p.config.BootstrapCommand == "" {
+		p.config.BootstrapCommand = "curl -s https://get.converge.sh | sh {{if ne .Version \"\"}}-s -- -v {{.Version}}{{end}}"
+	}
```

Comment:
I'm sceptical about that, on Ubuntu 14.04:

```
drwxr-xr-x 2 root root 4096 Nov 23 01:30 /usr/local/bin
```

Never the less, I would expect this software to be owned by `root` and if someone would like it some other way they can modify `boot_command` or use the `shell` provisioner to install it  any way they like.